packages/manager/apps/web/client/app/hosting/multisite/diagnostic/dialog.controller.js

```diff
@@ -1,34 +1,47 @@
-import { DIAGNOSTIC_STATE } from '../hosting-multisite.constants';
+import { RECORD_TYPE_TO_HOSTING_IP } from '../hosting-multisite.constants';
 
 export default class MultisiteDiagnosticCtrl {
   /* @ngInject */
-  constructor($scope, coreURLBuilder) {
-    this.scope = $scope;
+  constructor($scope, coreURLBuilder, HostingDomain, coreConfig) {
+    this.$scope = $scope;
```

Comment:
no need to store the scope as you don't use it in another place
```suggestion
```

Reply:
It's used in template for the reset action for close the modal

Reply:
copy all scope values just for one function seems a bit overkill. Why not using `.bind()` in your template ?
Moreover, I don't see any occurrence of `$ctrl.$scope` in the template, are you sure to use it ?